htdocs/mrp/class/mo.class.php

```diff
@@ -1869,6 +1869,11 @@ public function create(User $user, $notrigger = false)
 			return -1;
 		}
 
+		if (empty($this->fk_stock_movement) && $this->fields['disable_stock_change']['enabled'] && $this->role=='consumed') {
+			$this->error = 'StockChangeDisable';
```

Comment:
It seems there is a misunderstanting of what mean disable_stock_change
It means that we won't update the stock during the fabrication. It is not a lock.